### PR TITLE
Changed 'standard-deviation' to 'variance' in GroupNorm documentation

### LINKED ISSUE #141315: GroupNorm Doc Typo: Should Be Variance
### 📚 The doc issue

The [GroupNorm doc](https://pytorch.org/docs/stable/generated/torch.nn.GroupNorm.html) says
```
... The standard-deviation is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False).
```
It should be `variance`, rather than `standard-deviation`

### Suggest a potential alternative/fix

The ~~standard-deviation~~ variance `Var[x]` is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False).

cc @svekars @brycebortree @sekyondaMeta @AlannaBurke @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

### PR DESCRIPTION
**Pull request by Uvi-12**
Fixes #141315 

Updated the GroupNorm documentation to replace 'standard-deviation' with 'variance' to accurately reflect the calculation
method.

cc @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang @aakhundov

@pytorchbot label "topic: not user facing"

### CODE REVIEW

**Comment by Uvi-12** [comment]
@jbschlosser Please review the PR, It should solve the issue.

**Review by mikaylagawarecki — COMMENTED** [review]
Should this also apply to the other *{Norm} ops (e.g. LayerNorm) that use the same phrasing?

**Comment by Uvi-12** [comment]
@mikaylagawarecki Yes, this correction should apply to other operations if they use the same phrasing. Since the calculation is based on "variance," not "standard deviation," it's important to keep the documentation consistent. 

This PR solve the issue in the mentioned file.

**Comment by mikaylagawarecki** [comment]
Would you be okay to fix the other docs as well please

**Comment by Uvi-12** [comment]
> Would you be okay to fix the other docs as well please

Sure, I will fix the other docs too.

**Comment by Uvi-12** [comment]
@mikaylagawareck Fixed all docs to replace instances of 'standard-deviation' with 'variance'. Please review and merge the PR.

**Review by mikaylagawarecki — APPROVED** [review]
Thanks

**Comment by Uvi-12** [comment]
@mikaylagawarecki Can you please help me with the failing tests, it is not merged due to them.

**Comment by Uvi-12** [comment]
@mikaylagawarecki Maybe it requires approval for the workflows or you can directly merge the PR.

**Review by mikaylagawarecki — APPROVED** [review]
